docs/charter.md

```diff
@@ -1,7 +1,11 @@
 ICU4X Charter
 ==============
 
-ICU4X is a new project whose objective is to solve the needs of clients who wish to provide client-side i18n for their products in resource-constrained environments.
+ICU4X is a project whose objective is to solve the needs of clients who wish to provide client-side i18n for their products in resource-constrained environments.
+
+ICU4X derives from the experience of ICU4C/ICU4J and ECMA402 design decisions.
```

Comment:
ICU4X builds on the design and API decisions of ICU4C/ICU4X and ECMA-402?

(should use consistent naming for ECMA-402 …)

Also maybe endnote links to ICU[CJ], ECMA, etc…